src/keboola_mcp_server/clients/client.py

```diff
@@ -35,16 +40,28 @@ def get_metadata_property(
     :param metadata: A list of metadata entries.
     :param key: The metadata property key to search for.
     :param provider: Specifies the metadata provider name to filter by.
+    :param preferred_providers: Specifies a list of preferred metadata providers to order the metadata items by
```

Comment:
maybe we can add a check that `provider` and `preferred_providers` are not both set